gateway/ln-gateway/src/events.rs

```diff
@@ -0,0 +1,51 @@
+use fedimint_core::core::ModuleKind;
+use fedimint_core::Amount;
+use fedimint_eventlog::{Event, EventKind};
+use fedimint_lnv2_common::contracts::{Commitment, OutgoingContract};
+use serde::{Deserialize, Serialize};
+
+use crate::sha256::Hash;
+
+#[derive(Serialize, Deserialize)]
+pub struct OutgoingPaymentSuccess {
+    pub outgoing_contract: OutgoingContract,
+    pub min_contract_amount: Amount,
+    pub invoice_amount: Amount,
+    pub start_millis: u128,
+    pub end_millis: u128,
+    pub max_delay: u64,
+    pub is_swap: bool,
+}
+
+impl Event for OutgoingPaymentSuccess {
+    const MODULE: Option<ModuleKind> = Some(fedimint_lnv2_common::KIND);
```

Comment:
This events do not seem to be inside lnv2 module, so possibly it should be `None`. These are here mostly to avoid collisions between module authors naming their events the same by accident. 

Alternatively we should make up a face `ModuleKind` for `ln-gateway-core`.

Reply:
I made a fake `ModuleKind` for `ln-gateway-core`